### ISSUE TITLE
Closed campaigns are showing up in the finder results

### ISSUE THREAD

**Issue by mikefantini** [DoSomethingArchive/legacy-website#6374]
![image](https://cloud.githubusercontent.com/assets/6330971/14548325/dd65064e-0283-11e6-91f4-6dbf11aff9ed.png)

Dunk You Very Much and Everyday Superheroes are both closed.

**Comment by namimody** [comment]
Closed campaigns are also appearing in /campaigns as of 4:10 PM on 4/22:

![image](https://cloud.githubusercontent.com/assets/12396076/14753675/d1d8ac5a-08a4-11e6-958b-5f8ba4a0cc1e.png)

**Comment by namimody** [comment]
@angaither is this a Morgan or a Dave thing?

**Comment by angaither** [comment]
@blisteringherb?

**Comment by sergiitk** [comment]
This looks like a problem with index: see solr [query results](https://search.dosomething.org/solr/collection1/select?fq=-sm_field_campaign_status:%28closed%29%20bundle:[campaign%20TO%20campaign_group]%20ss_language:%28en-global%20OR%20en-global%29&wt=json&indent=false&facet=true&facet.field=fs_field_active_hours&facet.field=im_field_cause&facet.field=im_field_action_type&rows=1&bq=ss_language:en-global^10&group.field=entity_id&group=true&facet.query=im_field_cause:16&facet.query=im_field_cause:%2813%20OR%2017%29&facet.query=im_field_cause:12&facet.query=im_field_cause:14&facet.query=im_field_cause:2&facet.query=im_field_cause:4&facet.query=im_field_cause:%2819%20OR%205%29&facet.query=im_field_cause:%286%20OR%2015%29&facet.query=im_field_cause:%281%20OR%2021%29&facet.query=fs_field_active_hours:[*%20TO%201]&facet.query=fs_field_active_hours:[2%20TO%205]&facet.query=fs_field_active_hours:[6%20TO%20*]&facet.query=im_field_action_type:7&facet.query=im_field_action_type:3&facet.query=im_field_action_type:11&facet.query=im_field_action_type:8&facet.query=im_field_action_type:9&facet.query=im_field_action_type:18&facet.query=im_field_action_type:10&facet.query=im_field_action_type:20&start=0&q=_query_:%22{!func}scale%28is_bubble_factor,0,100%29%22%20AND%20fs_field_active_hours:%28%28%5B*%20TO%201%5D%29%29&json.wrf=solrResponse&fl=label,sm_field_campaign_status). 
![image](https://cloud.githubusercontent.com/assets/672669/14898650/82af1902-0d90-11e6-9a15-003216b985d0.png)

I'll reindex Solr and see if it's fixed.

**Comment by sergiitk** [comment]
> 517 items successfully processed. 606 documents successfully sent to Solr.

**Comment by sergiitk** [comment]
Hm, it's still marked as active.

**Comment by sergiitk** [comment]
So somehow [English](https://www.dosomething.org/us/node/6620/edit/en) translation is 'Closed', but [Global English](https://www.dosomething.org/node/6620/edit/en-global) is 'Active'.

**Comment by mikefantini** [comment]
Ah thanks for tracking this down @sergii-tkachenko. Flagging for @mnelson338 now.

**Comment by mikefantini** [comment]
@sergii-tkachenko sorry I misremembered a few things...

Yeah so ideally we could have a campaign active in one translation and closed in another. And so ideally if I'm on /us/campaigns it's checking to see what the status is for the English translation (if the campaign is translated in both US English and Global English; if it's just translated in Global English it should only check the status of Global English). And then on /campaigns (Global English version) ideally it would just check to see what the status is for the Global English translation.

Does that make sense? And is that possible?

**Comment by sergiitk** [comment]
Yes, this does make sense. Not sure if this is possible thought. Sounds like another thing to discuss in context of making #6394 work. What do you think, @namimody?
